www/site/content/docs/reference/config.md

```diff
@@ -299,83 +256,111 @@ Default
 : `true`
 
 Used by
-: Chef Workstation App
+: `chef-run`
 
-#### channel
+### Features
+
+Enable and disable experimental features for Chef Workstation.
+
+#### Example
+
+```toml
+[features]
+preferences_dialog = true
+analyze = true
+```
 
 Description
-: Set the update channel to use when checking for Chef Workstation updates. `"stable"` is the recommended value. Switch to `"current"` is not guaranteed to be stable, and should only be used if you are comfortable with the risks associated.
+: List of experimental features that can be turn on and off. In the above example, there are two turned on features, the `preferences_dialog` feature, and the `analyze` feature. Every feature flag has a way to be turned on through environment variables. The respective environment variable for the `analyze` feature would be `CHEF_FEAT_ANALYZE`. When set to any value, the feature will be turned on.
```

Comment:
The last sentence is confusing. Would this turn on both features?

```
[features]
preferences_dialog = true
analyze = false
```

Reply:
Yes. I guess the confusion is that I probably should have added both environment variable names and not only one? 🤔 

> Every feature flag has a way to be turned on through environment variables. The corresponding environment variable for the `analyze` feature would be `CHEF_FEAT_ANALYZE`, and for the `preferences_dialog` would be `CHEF_FEAT_PREFERENCES_DIALOG`. When set to any value, the feature will be turned on.

Reply:
No. In that snipped a user would have turned only one feature on.

Reply:
@kagarmoe I have updated this doc, could you give it another look please? 😄